automation/roles/patroni/tasks/main.yml

```
@@ -427,6 +427,22 @@
         state: directory
         mode: "0700"
 
+    # Generating TLS Certificates
+    - name: Generating TLS Certificates to {{ postgresql_home_dir }}/{{ postgresql_version }}
+      become: true
+      become_user: postgres
+      shell:
```

```suggestion
      ansible.builtin.command:
```
It is recommended to use the command module where it is not necessary to use a shell script